pandas/tests/plotting/test_misc.py

```diff
@@ -429,3 +429,96 @@ def test_dictionary_color(self):
         ax = df1.plot(kind="line", color=dic_color)
         colors = [rect.get_color() for rect in ax.get_lines()[0:2]]
         assert all(color == expected[index] for index, color in enumerate(colors))
+
+    def test_has_externally_shared_axis(self):
```

Comment:
how long does this test take? may need to mark it slow (we do this with most plotting functions)

Reply:
0.3-0.5s with over half of the runs being 0.33s

Reply:
Marked as slow.